src/seedu/addressbook/data/person/ReadOnlyPerson.java

```diff
@@ -82,4 +69,16 @@ default String getAsTextHidePrivate() {
         }
         return builder.toString();
     }
+    
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    default String getPrintableString(Printable...printables) {
+        final StringBuilder result = new StringBuilder();
+        for (int i = 0; i < printables.length; i++) {
+            result.append(printables[i]);
+            result.append("\n");
```

Comment:
Doesn't this result in an extra line break at the end? You could have used `StringJoiner` or `String.join` instead.